src/htr2hpc/train/README.md

```diff
@@ -0,0 +1,48 @@
+# How to use `htr2hpc.train`
+
+This submodule provides fucntionality for running HTR segmentation and transcription model training and fine-tuning tasks on an HPC system using content and models downloaded from an eScriptorium instance.
+
+The `htr2hpc-train` script will download document parts and optionally a model file and then start a slurm job to run the appropriate training task.
+
+## Setup
+
+On the HPC system (i.e., della for Princeton), create a new python 3.11 conda environment named `htr2hpc` and install the current version of this software:
+
+```sh
+module load anaconda3/2024.6
+wget https://github.com/mittagessen/kraken/raw/refs/heads/main/environment_cuda.yml
+conda env create -f environment_cuda.yml -n htr2hpc
+conda activate htr2hpc
+pip install git+https://github.com/Princeton-CDH/htr2hpc.git@feature/export-and-train-noparsl#egg=htr2hpc
```

Comment:
needs to be updated

Reply:
@cmroughan what directions should we use here? Should we replace with these?
```sh
module load anaconda3/2024.2
git clone https://github.com/mittagessen/kraken.git
cd kraken
conda env create -f environment_cuda.yml
conda activate kraken
pip3 install torchvision torch==2.1 torchaudio==2.1
```

I'm wondering how important it is to use the conda file from kraken since doing that requires cloning the repo, but would rather have a reliable working setup than spend time with problems caused by missing dependencies.

Reply:
I tested out an install where I followed the old directions (just the `pip install git+https://github.com/Princeton-CDH/htr2hpc.git@feature/export-and-train-noparsl#egg=htr2hpc`, no bothering with creating the env from kraken's `environment_cuda.yml` or with additional pip installs. The code did still run successfully, just with warnings about `No module named 'coremltools.libcoremlpython'`. I haven't gone back to the code yet to see where that is required, but maybe it's not the most necessary?

Otherwise, if we are sticking with using `environment_cuda.yml`, I think we can just keep those directions you laid out for now.